src/noteeditorlogic.h

```diff
@@ -35,7 +35,7 @@ class NoteEditorLogic : public QObject
 
     static QString getFirstLine(const QString &str);
     static QString getSecondLine(const QString &str);
-    void setTheme(Theme theme, QColor textColor);
+    void setTheme(Theme newTheme, QColor textColor, qreal fontSize);
```

Comment:
I'd prefer if we just kept calling this `theme` rather than `newTheme`. For consistency we could rename the occurrences of `newTheme` elsewhere to `theme` instead.